python/lsst/jointcal/astrometryTransform.cc

```diff
@@ -42,131 +37,159 @@ namespace lsst {
-    py::class_<AstrometryTransformSkyWcs, std::shared_ptr<AstrometryTransformSkyWcs>, AstrometryTransform>
-            cls(mod, "AstrometryTransformSkyWcs");
-    cls.def("getSkyWcs", &AstrometryTransformSkyWcs::getSkyWcs);
+void declareAstrometryTransformSkyWcs(lsst::cpputils::python::WrapperCollection &wrappers) {
+    using PyAstrometryTransformSkyWcs =
+            py::class_<AstrometryTransformSkyWcs, std::shared_ptr<AstrometryTransformSkyWcs>, AstrometryTransform>;
+
+    wrappers.wrapType(
+            PyAstrometryTransformSkyWcs(wrappers.module, "AstrometryTransformSkyWcs"), [](auto &mod, auto &cls) {
+                cls.def("getSkyWcs", &AstrometryTransformSkyWcs::getSkyWcs);
+            });
 }
 
-void declareBaseTanWcs(py::module &mod) {
-    py::class_<BaseTanWcs, std::shared_ptr<BaseTanWcs>, AstrometryTransform> cls(mod, "BaseTanWcs");
+void declareBaseTanWcs(lsst::cpputils::python::WrapperCollection &wrappers) {
+    using PyBaseTanWcs = py::class_<BaseTanWcs, std::shared_ptr<BaseTanWcs>, AstrometryTransform> ;
+
+    wrappers.wrapType(PyBaseTanWcs(wrappers.module, "BaseTanWcs"), [](auto &mod, auto &cls) {});
 }
 
-void declareTanPixelToRaDec(py::module &mod) {
-    py::class_<TanPixelToRaDec, std::shared_ptr<TanPixelToRaDec>, AstrometryTransform> cls(mod,
-                                                                                           "TanPixelToRaDec");
+void declareTanPixelToRaDec(lsst::cpputils::python::WrapperCollection &wrappers) {
+    using PyTanPixelToRaDec = py::class_<TanPixelToRaDec, std::shared_ptr<TanPixelToRaDec>, AstrometryTransform>;
+
+    wrappers.wrapType(PyTanPixelToRaDec(wrappers.module, "TanPixelToRaDec"), [](auto &mod, auto &cls) {});
 }
 
-void declareTanRaDecToPixel(py::module &mod) {
-    py::class_<TanRaDecToPixel, std::shared_ptr<TanRaDecToPixel>, AstrometryTransform> cls(mod,
-                                                                                           "TanRaDecToPixel");
+void declareTanRaDecToPixel(lsst::cpputils::python::WrapperCollection &wrappers) {
+    using PyTanRaDecToPixel = py::class_<TanRaDecToPixel, std::shared_ptr<TanRaDecToPixel>, AstrometryTransform>;
+
+    wrappers.wrapType(PyTanRaDecToPixel(wrappers.module, "TanRaDecToPixel"), [](auto &mod, auto &cls) {});
 }
 
-void declareTanSipPixelToRaDec(py::module &mod) {
-    py::class_<TanSipPixelToRaDec, std::shared_ptr<TanSipPixelToRaDec>, BaseTanWcs> cls(mod,
-                                                                                        "TanSipPixelToRaDec");
+void declareTanSipPixelToRaDec(lsst::cpputils::python::WrapperCollection &wrappers) {
+    using PyTanSipPixelToRaDec = py::class_<TanSipPixelToRaDec, std::shared_ptr<TanSipPixelToRaDec>, BaseTanWcs>;
+
+    wrappers.wrapType(PyTanSipPixelToRaDec(wrappers.module, "TanSipPixelToRaDec"), [](auto &mod, auto &cls) {});
 }
+}  // namespace
 
-PYBIND11_MODULE(astrometryTransform, mod) {
-    py::module::import("astshim.mapping");
```

Comment:
Does this need to be declared in `_jointcalLib.cc`?

Reply:
I don't see any dependencies on {{astshim.mapping}} here.